examples/docs/en-US/time-picker.md

```diff
@@ -94,14 +94,25 @@ If start time is picked at first, then the end time will change accordingly.
       minTime: startTime
     }">
   </el-time-select>
+  <el-time-picker
+    is-range
+    v-model="value4"
+    range-separator="To"
+    start-placeholder="Start time"
+    end-placeholder="End time"
+    :picker-options="{
+      selectableRange: ['17:30:00 To 18:30:00', '18:50:00 To 20:30:00', '21:00:00 To 22:00:00']
```

Comment:
Shouldn't `selectableRange` be `17:30:00 - 18:30:00`?
It's meant for developers.

Reply:
selectableRange is for developers. Better to avoid a particular language. 

‘A-B’ notion can be universally understood. End user won’t see selectableRange written in Vue code anyway. 

But “A to B” might not by meaningful to for example French developers. It also complicates parsing. “A-B” can just use string.split, but “A to B” may need to deal with space and words.